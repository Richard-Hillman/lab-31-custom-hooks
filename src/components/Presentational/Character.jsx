import React from 'react';
import PropTypes from 'prop-types';

function Character({ name, image }) {
  return (
    <>
      <figure>
        <img src={image} alt={name} />
        <figcaption>{name}</figcaption>    
      </figure>    
    </>
  );
}
Character.propTypes = {
  name: PropTypes.string,
  image: PropTypes.string
};
export default Character;
