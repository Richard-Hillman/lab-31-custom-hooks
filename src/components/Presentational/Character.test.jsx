import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Character from './Character';

describe('CharacterDetailsPresentation component', () => {
  afterEach(() => cleanup());
  it('renders Character', () => {
    const { asFragment } = render(<Character />);
    expect(asFragment()).toMatchSnapshot();
  });
});
