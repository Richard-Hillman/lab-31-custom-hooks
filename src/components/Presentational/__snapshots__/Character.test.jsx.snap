// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetailsPresentation component renders Character 1`] = `
<DocumentFragment>
  <figure>
    <img />
    <figcaption />
  </figure>
</DocumentFragment>
`;
